commons/constants.py

```diff
@@ -481,3 +481,8 @@ class Sizes:
 CORTX_ERR_DUPLICATE_BKT_NAME = ("An error occurred (BucketAlreadyOwnedByYou) when calling"
 "the CreateBucket operation: Your previous request to create the named bucket succeeded and "
 "you already own it.")
+RGW_ERR_NO_SUCH_UPLOAD = ("An error occurred (NoSuchUpload) when calling the UploadPart operation: Unknown")
+CORTX_ERR_NO_SUCH_UPLOAD = ("An error occurred (NoSuchUpload) when calling the UploadPart "
```

Comment:
Can we update the formatting of previous error messages strings to be consistent with this i.e. from lines 470 - 483

Reply:
sure